pkg/channel/get.go

```diff
@@ -17,18 +17,122 @@ limitations under the License.
 package channel
 
 import (
-	"github.com/bestchains/bc-cli/pkg/common"
+	"context"
+	"fmt"
+
 	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/kubectl/pkg/cmd/get"
+
+	"github.com/bestchains/bc-cli/pkg/common"
 )
 
+// NewChanGetCmd returns the `kubectl get` command for channel.
 func NewChanGetCmd(option common.Options) *cobra.Command {
-	//defaultPrintFlag := get.NewGetPrintFlags()
+	
+	defaultPrintFlag := get.NewGetPrintFlags()
+
 	cmd := &cobra.Command{
-		Use: "",
+		Use: "channel [NAME] -n NETWORK-NAME",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			// get config from getter
+			cfg, err := clientcmd.BuildConfigFromKubeconfigGetter("", common.InKubeGetter)
+			if err != nil {
+				fmt.Fprintln(option.ErrOut, err)
+				return err
+			}
+			cli, err := dynamic.NewForConfig(cfg)
+			if err != nil {
+				fmt.Fprintln(option.ErrOut, err)
+				return err
+			}
+			list := corev1.List{
+				TypeMeta: v1.TypeMeta{
+					Kind:       "List",
+					APIVersion: "v1",
+				},
+				ListMeta: v1.ListMeta{},
+			}
+			var obj runtime.Object
+			netName, _ := cmd.Flags().GetString("network")
+
+			if netName == "" {
```

Comment:
Set network as a required parameter.
```go
_ = cmd.MarkFlagRequired("network")
```

Reply:
`network` is set as a required parameter, the if judgment is not needed.
How to use the function `MarkFlagRequired` reference https://github.com/bestchains/bc-cli/blob/main/pkg/endorsepolicy/get.go#L174
setting `MarkFlagRequired` in RunE does not take effect.